lib/query-builders/inspector.js

```diff
@@ -16,6 +17,7 @@ module.exports = {
 
   inProgress: (queryBuilder, profile) => {
     queryBuilder.whereIn('status', [
+      ntcoEndorsed.id,
```

Comment:
I don't _think_ we need this change, but nothing bad happens by virtue of having it.